index.src.html

```diff
@@ -1049,6 +1062,20 @@ <h3 id="interfaces-credential-manager">Credential Manager</h3>
         credential: The credential to be stored.
       </pre>
     </dd>
+    <dt><dfn method>bad()</dfn></dt>
```

Comment:
@ithinkihaveacat suggested this in https://github.com/w3c/webappsec-credential-management/issues/56#issuecomment-274805979 as an option to leave the browser more choices for how to deal with the situation.

Let's discuss this in person tomorrow.